### PR TITLE
Feat/sector importing api

### PR DESCRIPTION
**Pull request by laser**
## What's in this PR?

This changeset reexports the proofs primitive operations [introduced here](https://github.com/filecoin-project/rust-fil-proofs-ffi/pull/34/files).

### CODE REVIEW

File: sector-builder-ffi/src/reexported.rs [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+            sector_id,
+            prover_id,
+            ticket,
+            comm_d,
+        )
+    })
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn sector_builder_ffi_reexported_destroy_verify_seal_response(
```
Comment (dignifiedquire) [inline sector-builder-ffi/src/reexported.rs]: why do you have to duplicate all of these? if this is parsed by cbindgen and they are reexported, they should show up

Comment (laser) [reply sector-builder-ffi/src/reexported.rs]: @dignifiedquire - 

> they should show up

They do not show up, which is why I reexport them.

Comment (dignifiedquire) [reply sector-builder-ffi/src/reexported.rs]: @vmx could you take a look at this when you have a moment, this seems really unfortunate

Comment (dignifiedquire) [reply sector-builder-ffi/src/reexported.rs]: (independent of this merging)

Comment (vmx) [reply sector-builder-ffi/src/reexported.rs]: *lol* I saw this conversation in my emails and thought "I need to take a look at that" and read the comment that I should have a look afterwards :)

Comment (vmx) [reply sector-builder-ffi/src/reexported.rs]: @laser @dignifiedquire It's not a cbindgen issue, but a Rust one (https://github.com/rust-lang/rfcs/issues/2771). I don't know much about linking C libraries, but as far as I understand, the problem is that those C functions are not re-exported, hence cbindgen can't pick them up. The only workaround is to wrap them manually, which is what this PR is doing.

Comment (laser) [reply sector-builder-ffi/src/reexported.rs]: Thanks for looking in to this, @vmx .

**Pull request merged**